### PR TITLE
Own dirs and files that dnf5 creates on runtime

### PR DESCRIPTION
**Pull request by tpgxyz**
Own dirs and files that dnf5 creates on runtime inside spec file so rpm can be aware of that fles

### CODE REVIEW

File: dnf5.spec [hunk 1]
```diff
@@ -172,14 +172,26 @@ It supports RPM packages, modulemd modules, and comps groups & environments.
...
+%dir %{_prefix}/lib/sysimage/dnf
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/groups.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/modules.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/nevras.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/packages.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/system.toml
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/transaction_history.sqlite
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/transaction_history.sqlite-shm
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/transaction_history.sqlite-wal
+%verify(not md5 size mtime) %ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/userinstalled.toml
```
Comment (m-blaha) [inline dnf5.spec:194]: I'd prefer to use a wildcard here: `%ghost %config(missingok,noreplace) %{_prefix}/lib/sysimage/dnf/*`

Comment (m-blaha) [reply dnf5.spec:194]: btw, is `%verify(not md5 size mtime)` needed for config files? (see https://src.fedoraproject.org/rpms/rpm/c/d58b662497ff7adc3be1682143ecb7f101ff1f71?branch=rawhide)

File: dnf5.spec [hunk 2]
```diff
@@ -535,6 +547,17 @@ DNF5 plugins.
 %install
 %cmake_install
 
+# own dirs and files that dnf5 creates on runtime
+mkdir -p %{buildroot}%{_prefix}/lib/sysimage/dnf
+for files in \
+    groups.toml modules.toml nevras.toml packages.toml \
+    system.toml transaction_history.sqlite \
+    transaction_history.sqlite-shm \
+    transaction_history.sqlite-wal userinstalled.tom
```
Comment (m-blaha) [inline dnf5.spec:556]: typo  - userinstalled.tom**l**

Comment (tpgxyz) [reply dnf5.spec]: Thanks for spotting this.

**Comment by m-blaha** [comment]
Can you rebase please? Your PR now conflicts with recent changes of dnf.spec file.

**Comment by tpgxyz** [comment]
@m-blaha Hi, sorry for the delay, hope i rebased it right.

**Comment by m-blaha** [comment]
Oh sorry - I wasn't available last week and now... another rebase is needed. Sorry for inconvenience again :(

**Comment by inknos** [comment]
@tpgxyz hi! 

> another rebase is needed. 

Could you please rebase the pr again?

@m-blaha I am taking over the review if you don't mind.

**Pull request merged**